server/models/Setting.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose')
+
+let SettingSchema = new mongoose.Schema({
+    username: {type: String, required: true},
+    option: {type: String, required: true},
+    modified: {type: Boolean, default: false}
+})
+
+module.exports = {
+    Setting: mongoose.model('Setting', SettingSchema)
```

Comment:
Because we're only exporting one thing we can just do `module.exports = mongoose.model('Setting', SettingSchema')` making it the default export